l10n_nl_intrastat/tests/test_all.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+from openerp.tests.common import TransactionCase
+
+
+class TestIntrastatNL(TransactionCase):
+    """Tests for this module"""
+
+    def setUp(self):
+        super(TestIntrastatNL, self).setUp()
+        self.tax = self.env['account.tax']
+        self.partner = self.env['res.partner']
+        self.intrastat_report = self.env['l10n_nl.report.intrastat']
+        self.account_receivable = self.env['account.account'].search(
+            [('user_type_id', '=',
+              self.env.ref('account.data_account_type_receivable').id)],
+            limit=1)
+        self.fiscal_position_model = self.env['account.fiscal.position']
+
+    def test_generate_report(self):
+        # Set our company's country to NL
+        company = self.env.ref('base.main_company')
+        company.country_id = self.env.ref('base.nl')
+
+        # Create a date range type
+        type = self.env['date.range.type'].create({
+            'name': 'Test date range type',
+            'company_id': company.id,
+            'allow_overlap': False
+        })
+
+        # Create a date range spanning the last three months
+        date_range = self.env['date.range'].create({
+            'name': 'FS2016',
+            'date_start': datetime.today() + relativedelta(months=-3),
+            'date_end': datetime.today(),
```

Comment:
Unit tests are failing in travis, could you check the [travis log](https://travis-ci.org/OCA/l10n-netherlands/jobs/207029684)?

I think the tests failure depends on these 2 lines: you are filling `date_start` and `date_end` with datetime objects. You should cast the datetime objects to `fields.Date` type instead, using `datetime.strftime()` for example. You already used `datetime.strftime()`, check in the model `l10n_nl.report.intrastat` for the example.